homeassistant/components/homekit/type_covers.py

```diff
@@ -334,6 +335,46 @@ def __init__(self, *args):
             *args, category=CATEGORY_WINDOW_COVERING, service=SERV_WINDOW_COVERING
         )
 
+@TYPES.register("Slat")
+class Slat(HomeAccessory):
+    """Generate a Slat accessory for a cover entity.
+
+    The entity must support: set_tilt_position.
+    """
+        
+    def __init__(self, *args):
+        super().__init__(*args, category=38) # https://docs.microsoft.com/en-us/dotnet/api/homekit.hmservicetype?view=xamarin-ios-sdk-12
+
+        serv_slat = self.add_preload_service('Slat', ["SlatType", "CurrentSlatState", "CurrentTiltAngle", "TargetTiltAngle"])
```

Comment:
https://github.com/ikalchev/HAP-python/blob/dev/pyhap/resources/services.json#L414

Reply:
Looking at my implementation, I am implementing the right characteristics? I am afraid that the 'Slat' service is not supported in HomeKit (yet)... I couldn't find any implementation of it and it is only mentioned in the Apple HAP PDF.